lib/modules/RESTips.js

```diff
@@ -11,9 +11,9 @@ import * as SettingsNavigation from './settingsNavigation';
 export const module = {};
 
 module.moduleID = 'RESTips';
-module.moduleName = 'RES Tips and Tricks';
-module.category = 'About RES';
-module.description = 'Adds tips/tricks help to RES console';
+module.moduleName = 'resTipsName';
+module.category = 'resTipsCategory';
```

Comment:
We can't do this because the category needs to be the same for each module that's in the same category. In general, if there are multiple strings that are the same, they should be translated with the same key, so that stuff behaves the same regardless of whether or not it's translated.

It should instead be `aboutResCategory`, `myAccountCategory`, `appearanceCategory`, etc.

You'll also need to change `categories` in metadata.js to the untranslated names so that category sorting will work correctly